ui/raidboss/data/07-dt/raid/r6s.ts

```diff
@@ -180,9 +180,10 @@ const triggerSet: TriggerSet<Data> = {
       run: (data, matches) => data.lastDoubleStyle = doubleStyleMap[matches.id],
     },
     {
+      // tether source is from the actor to the boss
       id: 'R6S Double Style Tether Tracker',
       type: 'Tether',
-      netRegex: { targetId: '4[0-9A-Fa-f]{7}', id: ['013F', '0140'], capture: true },
+      netRegex: { sourceId: '4[0-9A-Fa-f]{7}', id: ['013F', '0140'], capture: true },
```

Comment:
IIRC I went with `targetId` here instead of `sourceId` to allow also matching against the normal/wing bomb tether at the end of desert phase (since `sourceId` entries are player IDs for that mechanic), but that's being handled by a different trigger now so it shouldn't matter.

Reply:
I changed it here because like you said, we have another trigger handling the desert phase player tethers, and this one was collecting unnecessary data then.